### PR TITLE
Add style corrections to comply with PEP8

### PR DESCRIPTION
**Pull request by vnmabus**
* Remove bare except.
* PEP8 line limit enforced.
* Unify some urls to use https.

### CODE REVIEW

File: skdatasets/keel.py [hunk 1]
```diff
@@ -95,189 +103,309 @@ def _fetch(name, url, names, target_name, nfolds=5):
...
+        'args': [BASE_URL +
+                 'keel-dataset/datasets/imbalanced/imb_IRhigherThan9p1',
+                 ['Mcg', 'Gvh', 'Lip', 'Chg', 'Aac',
+                  'Alm1', 'Alm2', 'Class'],
+                 'Class']
+        },
+    'ecoli-0-1-4-6_vs_5': {
+        'args': [BASE_URL +
+                 'keel-dataset/datasets/imbalanced/imb_IRhigherThan9p2',
+                 ['a1', 'a2', 'a3', 'a5', 'a6',
```
Comment (vnmabus) [inline skdatasets/keel.py]: Is there any reason for the missing 'a4' in some ecoli datasets?

Comment (daviddiazvico) [reply skdatasets/keel.py]: They are built that way: https://sci2s.ugr.es/keel/dataset.php?cod=981

**Comment by daviddiazvico** [comment]
Good. Aside from the discussion on the ordering of the imports I just opened, I want to ask if you have tested the new URLs with https. If they work and all datasets can be fetched correctly, then I think this is ready to merge.

**Comment by vnmabus** [comment]
> 
> 
> Good. Aside from the discussion on the ordering of the imports I just opened, I want to ask if you have tested the new URLs with https. If they work and all datasets can be fetched correctly, then I think this is ready to merge.

I tested the a4a dataset, as an example of a libsvm dataset that was originally accessed using http. It worked just fine with https and I assumed that all libsvm datasets would work then.

I tried testing the UCI datasets, but the UCI server was down.

keel datasets did not work using https, so they remain with http.

**Comment by vnmabus** [comment]
Ok, as the UCI servers are up again, I have tried accessing the nursery dataset using https without trouble.

**Pull request merged**